ui/app/pages/send/send.utils.js

```diff
@@ -243,12 +244,17 @@ async function estimateGas ({
   }
 
   // if not, fall back to block gasLimit
+  if (!blockGasLimit) {
+    blockGasLimit = ARBITRARY_HIGH_BLOCK_GAS_LIMIT
```

Comment:
I'm not _entirely_ sure how the problem this is solving (the account tracker not having a valid current block gas limit) relates to the bug this PR is meant to address. That problem might also be better solved in the background, though I'm not sure how... maybe by having a valid default instead of empty string, and/or exposing the 'loading' state to the UI so it can wait for the first block to get read, and/or dealing with errors in a different manner.

I suppose setting this default here for the time being is fine though; maybe it's the quickest way to fix this for now.

Reply:
Oh, yeah, sorry. This solves an additional bug that was discovered while QAing the fix to the primary bug. It relates in the sense that it is a bug in our gas estimation logic.

I would say a background fix may also be needed, but as it stands, if this function receives a falsey but defined `blockGasLimit`, it will break as such a value will cause an error when passed to `multiplyCurrencies` below.